modules-detection.mjs

```diff
@@ -14,7 +14,8 @@ async function detect() {
         return;
     }
 
-    const { stdout: fileChangesRaw } = await $`git diff --name-only  ${sourceRef}..${targetRef}`;
+    await $`git fetch`;
```

Comment:
`git fetch` may take a few seconds.. is it worth it?

Reply:
Yes. I didn't find any other options to tell git which branches exists